### PR TITLE
Content formatting updates

### PR DESCRIPTION
**Pull request by SteveLasker**
Several updates to:

- make it easier to follow the samples
- markdown linting fixes
- python version reference update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,111 +29,148 @@ conda env create -f environment.yml
...
+1. Start the service, under the `/workspace` directory, using CCF
 
-```sh
-./scitt-emulator.sh server --workspace workspace/ --tree-alg RKVST
-```
+    ```sh
+    ./scitt-emulator.sh server --workspace workspace/ --tree-alg CCF
+    ```
+
+1. The server is running at http://localhost:8000/ and uses the `/workspace` folder to store the service parameters and service state
```
Comment (aj-stein-nist) [inline README.md]: Minor-nit: although I understand what you mean, writing `/workspace` with the leading slash implies an absolute path from the root of whatever filesystem the emulator code is cloned to, did you mean `workspace/` given the above command?

```suggestion
1. The server is running at http://localhost:8000/ and uses the `workspace/` folder to store the service parameters and service state  
```

File: README.md [hunk 2]
```diff
@@ -145,30 +184,40 @@ The `service_parameters.json` file gets created when starting a service using `.
...
+`scitt_emulator/ccf.py` is the implementation of the [CCF tree algorithm](https://ietf-scitt.github.io/draft-birkholz-scitt-receipts/draft-birkholz-scitt-receipts.html#name-ccf-tree-algorithm).
+For each claim, a receipt is generated using a fake but valid Merkle tree that is independent of other submitted claims.
+A real CCF service would maintain a single Merkle tree covering all submitted claims and auxiliary entries.
 
-`scitt_emulator/rkvst.py` is a simple REST proxy that takes SCITT standard API calls and routes them through to the [RKVST production SaaS service](https://app.rkvst.io). Each claim is stored in a Merkle tree underpinning a Quorum blockchain and  receipts contain valid, verifiable inclusion proofs for the claim in that Merkle proof. [More docs on receipts here](https://docs.rkvst.com/platform/overview/scitt-receipts/).
+`scitt_emulator/rkvst.py` is a simple REST proxy that takes SCITT standard API calls and routes them through to the [RKVST production SaaS service](https://app.rkvst.io).
+Each claim is stored in a Merkle tree underpinning a Quorum blockchain and  receipts contain valid, verifiable inclusion proofs for the claim in that Merkle proof.
```
Comment (aj-stein-nist) [inline README.md]: Minor nit: double space between "and" and "receipts."

```suggestion
Each claim is stored in a Merkle tree underpinning a Quorum blockchain and receipts contain valid, verifiable inclusion proofs for the claim in that Merkle proof.
```

File: docs/registration_policies.md [hunk 3]
```diff
@@ -140,21 +140,21 @@ echo ${CLAIM_PATH}
...
-$ scitt-emulator server --workspace workspace/ --tree-alg CCF --use-lro
+scitt-emulator server --workspace workspace/ --tree-alg CCF --use-lro
 ```
 
 Create claim from allowed issuer (`.org`) and from non-allowed (`.com`).
 
 ```console
 $ scitt-emulator client create-claim --issuer did:web:example.com --content-type application/json --payload '{"sun": "yellow"}' --out claim.cose
-Claim written to claim.cose
+A COSE signed Claim was written to:  claim.cose
```
Comment (aj-stein-nist) [inline docs/registration_policies.md]: Minor-nit: I won't repeat in the other five instances, but do we mean to hyphenate this or in the upstream IETF documents that is not done?

```suggestion
A COSE-signed Claim was written to:  claim.cose
```

**Review by aj-stein-nist — COMMENTED** [review]
Minor nits but otherwise looks good. (I am new to the community and cannot contribute or would otherwise approve with minor typo feedback and be more clear about my support despite the nits.)

**Comment by SteveLasker** [comment]
@aj-stein-nist, thanks for the reviews and feedback. All were merged. 
As for becoming an active member, you're doing great and we really appreciate the engagement in whatever capacity you can provide.

**Comment by SteveLasker** [comment]
BTW, a non-member/non-binding LGTM is still appreciated and considered. Especially, this early in the process.

**Review by OR13 — APPROVED** [review]

**Pull request merged**